http/base/rest/interfaces.go

```diff
@@ -565,6 +565,36 @@ func GetShardTxState(cmd map[string]interface{}) map[string]interface{} {
 	return resp
 }
 
+func GetShardChainConfig(cmd map[string]interface{}) map[string]interface{} {
+	resp := ResponsePack(berr.SUCCESS)
+	idStr, ok := cmd["ShardID"].(string)
```

Comment:
use uint64 to replace string while type assert

Reply:
no,can not use uint64,only support string in  rest interface parsm